### ISSUE TITLE
Scrollbar Hidden not working

### ISSUE THREAD

**Issue by Gabb1995** [bfwg/ngx-drag-scroll#250]
```html
<drag-scroll #dragscroll *ngIf="thumbnails && thumbnails.length !== 0" style="height: 100%" [scrollbar-hidden]="true">
     <app-thumbnail (click)="wlService.scrollToRegistration()" *ngFor="let thumbnail of thumbnails"
          [model]="thumbnail" drag-scroll-item></app-thumbnail>
 </drag-scroll>
```

I added the option `scrollbar-hidden` but I still see the scrollbar at the bottom on desktop devices. When I inspect element I see that `drag-scroll-content` div has the attribute `overflow: auto`.

To fix it myself I had to do:

```css
.drag-scroll-content {
  overflow: hidden !important;
}
```

which is not nice. Is it possible to let me know if im doing something wrong?

**Comment by thomascayne** [comment]
I agree with @Gabb1995 I am creating a scrollable tab, where the tab items (title bar) shows buttons to the left/right. There was a blank space where the scrollbar was which pushing the tab content down. The following removed the empty space where the scrollbar was without affecting functionality of ng-drag-scroll.

```
.drag-scroll-content {
  overflow: hidden !important;
}

.drag-scroll-container {
  height: auto !important;
  .drag-scroll-content {
    height: auto !important;
    overflow: hidden !important;
    width: auto !important;
  }
}

```

**Comment by yociduo** [comment]
The root cause is that the component can't get the event when the child node width and height change. For example, the child component is an image, After image loading, the image will be loaded as the actual size, but the component can't the real size when initialize.

**Comment by Thomas1664** [comment]
I have the same problem and I am unable to override it:
```
<div _ngcontent-nfc-c12="" class="drag-scroll-content" drag-scroll="true" style="width: 100%; height: 100%; overflow: auto hidden; display: block; white-space: nowrap;"></div>
```

Please fix it!!!

**Comment by bfwg** [comment]
Hey @Thomas1664 , which version of the library are you using? Have you tried the latest release?

**Comment by Thomas1664** [comment]
> Hey @Thomas1664 , which version of the library are you using? Have you tried the latest release?
package.json:
```
"ngx-drag-scroll": "^9.0.0-beta.3"
```

**Comment by bfwg** [comment]
I'm having a hard time reproducing the issue on my end. I would be grateful if you can create a https://plnkr.co/ with your use case so I can work the fix form there.

**Comment by Thomas1664** [comment]
the problem is that although I disabled scrollbars, ```.drag-scroll-content``` has ```overflow-x: auto```. But is should be hidden since scrollbars are disabled.

```html
<drag-scroll _ngcontent-vxv-c54="" class="svg-draggable" _nghost-vxv-c12="" ng-reflect-scrollbar-hidden="true"
  ng-reflect-y-disabled="true" style="pointer-events: auto;">
  <div _ngcontent-vxv-c12="" class="drag-scroll-wrapper drag-scroll-container"
    style="width: 100%; height: 124px; overflow: hidden;">
    <div _ngcontent-vxv-c12="" class="drag-scroll-content" drag-scroll="true"
      style="width: 100%; height: 100%; overflow: auto hidden; display: block; white-space: nowrap;">
      <div _ngcontent-vxv-c54="" style="width: 100%">
      </div>
    </div>
  </div>
</drag-scroll>
```

I think I have a solution for this problem:

The method [enableScrolling](https://github.com/bfwg/ngx-drag-scroll/blob/1c31560e5b3eb5fce7123127203f49e16d3ce848/projects/ngx-drag-scroll/src/lib/ngx-drag-scroll.component.ts#L487) in ngx-drag-scroll.component.ts needs to check if scrollbars are enabled or not. Currently, the method looks like this: 

```typescript
private enableScroll(axis: string): void {
    this._renderer.setStyle(this._contentRef.nativeElement, `overflow-${axis}`, 'auto');
  }
```

If you're fine with my suggestion I would create a PR and apply the fix.

**Comment by DanteDeRuwe** [comment]
Have the same issue
No scrollbars, but if you zoom in and out they're there. sometimes they show up as lines randomly

**Comment by bfwg** [comment]
Yes please @Thomas1664
